cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help

```diff
@@ -3,40 +3,31 @@ NAME
 
 SYNOPSIS
        ballerina add <module-name> [-t <template-name> | --template <template-name>]
-       ballerina add -l | --list
-
 
 DESCRIPTION
        Add creates a new directory and initializes a Ballerina module inside
-       the 'src' directory of the project. The command should be run inside
+       the 'modules' directory of the project. The command should be run inside
        the project root.
 
        A template can also be specified when creating the new module.
        If a template is not specified, this command creates a module
-       with a 'main' function.
-
-       There are two built-in templates named: 'main' and 'service'.
+       with a 'lib' template.
 
        You can download templates from Ballerina Central as well.
        In that case <template-name> becomes a qualified module name.
 
-           <org-name>/<module-name>[:<version>]
+           <package-name>/<module-name>[:<version>]
```

Comment:
we need the org name as well. Lets not change it till we finalize the Central API